tests/cli/test_cmds_status.py

```diff
@@ -119,19 +123,19 @@ def test_report_status_for_cluster_displays_deployed_service(
         % (fake_status)
     )
 
-    status.report_status_for_cluster(
+    _, output = status.report_status_for_cluster(
         service=service,
         cluster='fake_cluster',
         deploy_pipeline=planned_deployments,
         actual_deployments=actual_deployments,
         instance_whitelist=instance_whitelist,
         system_paasta_config=system_paasta_config,
     )
-    output, _ = capfd.readouterr()
```

Comment:
Does this test (and subsequent ones) still need the capfd fixture after you remove this?